src/components/atoms/Timeline/index.tsx

```diff
@@ -159,23 +204,31 @@ const MemoryList: React.FC<MemoryListProps> = memo(
           }
           return <Memory key={`${idx}`} isHour={isHour} isStrongMemory={isStrongMemory} />;
         })}
-      </MemoryContainer>
+      </>
     );
   },
-  (prev, next) => prev.memoryCount === next.memoryCount && prev.start === next.start,
+  (prev, next) =>
+    prev.memoryCount === next.memoryCount &&
+    prev.start === next.start &&
+    prev.memoryInterval === next.memoryInterval &&
+    prev.strongMemoryHours === next.strongMemoryHours,
 );
 
 const Container = styled.div`
   background: ${({ theme }) => theme.colors.publish.dark.other.background};
   width: 100%;
   display: flex;
   align-items: center;
-  padding: ${({ theme }) => `${theme.metrics.m} ${theme.metrics.xs}`};
+  padding: ${({ theme }) => `${theme.metrics.m}px ${theme.metrics.xs}px`};
   box-sizing: border-box;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
 `;
 
 const MemoryBox = styled.div`
-  border: 1px solid ${({ theme }) => theme.colors.publish.dark.icon.weak};
+  border: ${BORDER_WIDTH}px solid ${({ theme }) => theme.colors.publish.dark.icon.weak};
```

Comment:
This might be a personal preference, so you don't have to do this, but I feel like it is easier to read like this:
```
border: ${({ theme }) => `${BORDER_WIDTH}px solid ${theme.colors.publish.dark.icon.weak}`};
```
That way, the structure of the border value is upheld and not broken up as much. What do you think?

Reply:
I fixed! [refactor: border style](https://github.com/reearth/reearth-web/pull/264/commits/d5d30090118b411b25aae0b9bacaf493b28a9678)